services/Mumble/index.ts

```diff
@@ -63,8 +63,13 @@ export class MumbleService {
       }
 
       const response = await fetch(`${this.baseUrl}/${path}`, options);
+
+      // all but 200s are going to be rejected?
```

Comment:
Who is this question for? Perhaps we could create an issue and discuss it

Reply:
for me. I will check at this a bit and if have still thoughts about it, we can discuss it and open an issue. 👍🏽 
For the moment I will remove it.